app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    pass
```

Comment:
I disagree, because we can see the text of the error in the file where it is called, it is more convenient than going to another file to read or edit them.
But I changed it according to the comment